### PR TITLE
Reformat Air Gap Doc for gaps/printing

### PR DESCRIPTION
**Pull request by MBishop17**
Still in progress, but I wanted to get some feedback rolling for the single node air gap install. Please review `/installation/air-gap-installation/single-node/_index.md`.

### CODE REVIEW

File: content/rancher/v2.x/en/installation/air-gap-installation/single-node/_index.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+If you are installing Rancher in a development or testing environment where identity verification isn't a concern, install Rancher using the self-signed certificate that it generates. This installation option omits the hassle of generating a certificate yourself.
+
+Log into your Linux host, and then run the installation command below. Replace `<REGISTRY.YOURDOMAIN.COM:PORT>` with your private registry URL and port.
+
+    docker run -d --restart=unless-stopped \
+    -p 80:80 -p 443:443 \
+    <REGISTRY.YOURDOMAIN.COM:PORT>/rancher/rancher:<RANCHER_VERSION_TAG>
+
+{{% /accordion %}}
+{{% accordion id="option-b" label="Option B-Bring Your Own Certificate: Self-Signed" %}}
```
Comment (MBishop17) [inline content/rancher/v2.x/en/installation/air-gap-installation/single-node/_index.md]: Question: Can you use a cert signed by a public CA (e.g. digicert or another company) in an air gap config? Nodes have to have internet access to validate the cert, right?

File: content/rancher/v2.x/en/installation/air-gap-installation/single-node/_index.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+## 2. Collect Image Sources
+
+Using a computer with internet access, browse to our Rancher [releases page](https://github.com/rancher/rancher/releases) and find the version that you want to install. Download the following three files, which are required to install Rancher in an air gap environment:
+
+
+| Release File | Description |
+| --- | --- |
+| `rancher-images.txt` | This file contains a list of all files needed to install Rancher.
+| `rancher-save-images.sh` | This script pulls all the images in the `rancher-images.txt` from various public registries and saves all of the images as `rancher-images.tar.gz`. |
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/installation/air-gap-installation/single-node/_index.md]: I think it's only pulling images from Dockerhub, and not any other public registry.

File: content/rancher/v2.x/en/installation/air-gap-installation/single-node/_index.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+As of v2.1.0, registries with authentication work for installing Rancher as well as provisioning clusters.
+
+
+## 1. Provision Linux Host
+
+Provision a single, air gapped Linux host according to our [Requirements]({{< baseurl >}}/rancher/v2.x/en/installation/requirements) to launch your {{< product >}} Server.
+
+This host should be disconnected from the internet, but should have connectivity with your private registry.
+
+## 2. Collect Image Sources
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/installation/air-gap-installation/single-node/_index.md]: We should merge step 2 and 3 together as `Populate Your Private Registry`. I don't believe when we're downloading these artifacts we're actually `collecting image sources`.

File: content/rancher/v2.x/en/installation/air-gap-installation/single-node/_index.md (hunk 1 above)
Comment (deniseschannon) [reply content/rancher/v2.x/en/installation/air-gap-installation/single-node/_index.md]: can you follow up with @jgreat with this question?

File: content/rancher/v2.x/en/installation/air-gap-high-availability/provision-hosts/_index.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+<figcaption>HA Rancher install with layer 4 load balancer, depicting SSL termination at ingress controllers</figcaption>
+
+![Rancher HA]({{< baseurl >}}/img/rancher/ha/rancher2ha.svg)
+
+### Load Balancer
+
+RKE, the installer that provisions your air gapped cluster, will configure an Ingress controller pod on each of your nodes. The Ingress controller pods are bound to ports TCP/80 and TCP/443 on the host network and are the entry point for HTTPS traffic to the Rancher server.
+
+Configure a load balancer as a basic Layer 4 TCP forwarder. The exact configuration will vary depending on your environment.
```
Comment (MBishop17) [inline content/rancher/v2.x/en/installation/air-gap-high-availability/provision-hosts/_index.md]: @jgreat , any special config requirements for an air gap load balancer? If not, I'm going to link back to the aws example config.

File: content/rancher/v2.x/en/installation/air-gap-high-availability/prepare-private-registry/_index.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+| Release File | Description |
+| --- | --- |
+| `rancher-images.txt` | Contains a list of all files needed to install Rancher.
+| `rancher-save-images.sh` | Pulls all the images in the `rancher-images.txt` from various public registries and saves all of the images as `rancher-images.tar.gz`. |
+| `rancher-load-images.sh` | Loads images from the `rancher-images.tar.gz` file and pushes them to your private registry. |
+
+
+The Rancher HA install uses images from 3 sources. Combine the 3 sources into a file named `rancher-images.txt`.
+
+* **Rancher** - Images required by Rancher. Download the `rancher-images.txt` file from [Rancher releases](https://github.com/rancher/rancher/releases) page for the version of Rancher you are installing.
```
Comment (MBishop17) [inline content/rancher/v2.x/en/installation/air-gap-high-availability/prepare-private-registry/_index.md]: @jgreat, are the commands in this list steps that you need to complete before continuing? I'm not really clear at first read.

**Pull request merged**